### ISSUE TITLE
Incorrect generated authorization header with private client type

### ISSUE THREAD

**Issue by junwen-k** [florianmartens/passport-twitter-oauth2.0#2]
Thank you for writing this passport.

I've noticed that the generated Authorization header is not working as expected.

```js
const base64url = require('base64url');

const username = 'user';
const password = 'password';

console.log('Basic ' + base64url(`${username}:${password}`)); // Method implemented in this passport.
console.log('Basic ' + Buffer.from(`${username}:${password}`).toString('base64'));
```

Notice that the generated authorization header is different. Is this intentional or am I missing something? Thank you.

**Comment by florianmartens** [comment]
Oh sorry, it has been a minute since I checked this. But, you're right. The twitter api expects a base64 encoded string; not a base64-url-encoded string. This has not produced any errors on my end though. But I will address this soon.

**Comment by florianmartens** [comment]
Closed #3
